core/rpc/src/Mu/Rpc/Examples.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE TupleSections #-}
```

Comment:
Lowercase here 😝

Reply:
Aaargh, ok, I need that automatic formatter you've talked me about